src/enhance.ts

```diff
@@ -1,6 +1,6 @@
 import 'core-js/stable';
 import 'regenerator-runtime/runtime';
-import { uniq } from 'lodash';
+import { initial, uniq } from 'lodash';
```

Comment:
please use the esm version of lodash, ‘lodash-es’

Reply:
![image](https://user-images.githubusercontent.com/18328465/114868132-459b6d00-9dfe-11eb-8849-9efba516a8f0.png)
a bundler is needed to load lodash-es -> talked with Georges and we'll stay on lodash for this.